subprojects/core/src/main/java/org/gradle/api/internal/tasks/execution/ExecuteActionsTaskExecuter.java

```diff
@@ -120,7 +120,7 @@ private void executeAction(final String actionDisplayName, final TaskInternal ta
         buildOperationExecutor.run(new RunnableBuildOperation() {
             @Override
             public BuildOperationDescriptor.Builder description() {
-                return BuildOperationDescriptor.displayName(actionDisplayName + " for " + task.getPath()).name(actionDisplayName);
```

Comment:
fixed this as stumbling upon this when evaluating task execution build operation events for composite builds